paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -157,6 +157,14 @@ internal fun SelectPaymentMethod(
                 )
             }
         }
+
+        AnimatedVisibility(visible = viewState.mandateText != null) {
+            Mandate(
```

Comment:
I believe we’re showing this centered in other screens now. Do we need to make any changes here, given that it renders as left-aligned?

Reply:
Sorry what do you mean? It is rendering in the center. Is it supposed to be left aligned?

Reply:
The [screenshots](https://github.com/stripe/stripe-android/pull/7595/files#diff-01664d6394e22425148067d045f7abf5038014f9c3437132a861961b30885d3f) render it left-aligned right now.

Reply:
Ohh strange... ok thank you, I will take a look.

Reply:
@tillh-stripe fixed, it was because the text length is too short.

Reply:
That’s not the proper fix. We need to force the mandate text composable to occupy the full width. Only then does `TextAlign.Center` do what it’s supposed to do.